tensorflow_2/README.md

```diff
@@ -0,0 +1,45 @@
+# Samples for Notebook to Kubeflow Deployment using TensorFlow 2.0 Keras
+This directory aims at building Kubeflow use case samples with Tensorflow 2.0 Keras training code demonstrating 'Customer User Journey' (CUJ) in the process.
+
+The samples hosted are as follows - 
+### Text Classification 
+Based on a [Tensorflow tutorial](https://www.tensorflow.org/tutorials/keras/text_classification_with_hub), this machine learning task aims at classifying IMDB movie 
+reviews. Building on this, this sample hosts the following,
```

Comment:
What does "Building on this" mean? Also same comments with writing style

Reply:
"Building on this" means we took tensorflow tutorial code as our training code and then demonstrated Kubeflow functionalities on it

Reply:
Probably need to rephrase it so it reads better in proper English.

Reply:
Rephrased all notebook details to be more readable.